### PR TITLE
fix(DB/SmartAI): Unliving Stalker Viper Sting heroic only

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  Unliving Stalker casts Viper Sting in heroic mode only

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13784

## Tests Performed:

- tested on local

## How to Test the Changes:

1..tele ac
2. Unliving Stalker is called by Auchenai Soulpriest/Monk/Vindicator, of which there are several at the beginning of the dungeon. But they have several types of NPCs to summon, so you may need to kill them and .resp

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nefertumm — CHANGES_REQUESTED** [review]
The filename can't contain spaces

**Review by Nefertumm — APPROVED** [review]

File: data/sql/updates/pending_db_world/heroicViperSting.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Set Viper Sting to only in heroic
+Update `smart_scripts` set `event_flags`=4 WHERE (`entryorguid` = 18501) AND (`source_type` = 0) AND (`id` IN (4));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/heroicViperSting.sql:2]: Always use Versals for SQL Keywords, such as UPDATE and SET

**Review by Kitzunu — CHANGES_REQUESTED** [review]
above

**Comment by BeefSteaker** [comment]
my bad, client usually does that automatically, must have changed this one manually

**Pull request merged**